### ISSUE TITLE
Docs: How-to - Dbt packages

### ISSUE THREAD

**Issue by bmtcril** [openedx/openedx-aspects#73]

**Comment by pomegranited** [comment]
@bmtcril  Does https://github.com/openedx/openedx-aspects/pull/111 address this issue sufficiently?

I'm happy to expand on the [Extending dbt](https://docsopenedxorg--111.org.readthedocs.build/projects/openedx-aspects/en/111/how-tos/dbt_extensions.html) page if we want to provide a more detailed tutorial.

**Comment by bmtcril** [comment]
@pomegranited I think we may want a step-by-step for that, unless @Ian2012 has already covered that elsewhere or has a starter doc?

**Comment by Ian2012** [comment]
@pomegranited That's the only thing we need to expand DBT, for the conference we can include a video tutorial and a demo package that includes some useful models for custom stuff. I can consider this as closed

**Comment by pomegranited** [comment]
@Ian2012  Cool, I think a demo package and a tutorial will help people, and it would help me to go through the whole process to understand it better.

But I don't think we should include a video -- they're not very accessible and are hard to translate. Would a step-by-step tutorial on this docs site be ok?

**Comment by Ian2012** [comment]
Yes, sure

**Comment by pomegranited** [comment]
@bmtcril @Ian2012 I have hit an issue with dbt that I can't figure out from the docs.. can you help? Here's what happened:

1. Created this [aspects-dbt-example @ jill/demo](https://github.com/open-craft/aspects-dbt-example/compare/jill/demo) and configured my tutor dev stack to use it.
2. The first time I ran it (using `tutor dev do dbt -c "run"`), I used the wrong schema + table name: [xapi.problem_events_mv](https://github.com/open-craft/aspects-dbt-example/compare/jill/demo?expand=1#diff-479d8e6025b0ff04416d13d22d1f5adb805682b3fa4d0175d6f3fbe61a4c30cbR9) was incorrectly written as `reporting.problem_events`, and so dbt accurately reported the error:

   ```
   06:05:28    Database Error in model average_attempts (models/problem_responses/average_attempts.sql)
   :HTTPDriver for http://clickhouse:8123 returned response code 404)
   Code: 60. DB::Exception: Table reporting.problem_events does not exist. (UNKNOWN_TABLE) (version 23.8.9.54 (official build))
   compiled Code at target/run/aspects_dbt_example/models/problem_responses/average_attempts.sql
   ```
3. I fixed the issue in the model, but I can't get rid of the error!

   Clickhouse seems to have stored the relationship; I can see this line at the top of this file `$(tutor config printroot)/data/clickhouse/store/e4a/e4a29e0d-0e2f-44fb-9e2b-0d62549aaa57/problem_events_mv.sql`:
   ```
   ATTACH MATERIALIZED VIEW _ UUID 'e25be6d2-1ed2-481a-a47b-457b6a989c6f' TO reporting.problem_events`
   ```
 
I've tried `dbt clean`, and adding `[+](https://docs.getdbt.com/reference/resource-configs/plus-prefix)[full_refresh](https://docs.getdbt.com/reference/resource-configs/full_refresh): true` to the model config in `dbt_project.yml`, but neither helped.

This is just my devstack, so it's no big deal to blow away and start again. But since I'm writing a tutorial on how to create a custom dbt package, it would be really nice to resolve this without extreme measures!

**Comment by bmtcril** [comment]
@pomegranited I think you have a version with the dbt bug that didn't get fixed until 0.73.1 yesterday. Then the build broke on those changes, so you'd have to build aspects / aspects-superset yourself until the next fix goes out, hopefully today. Sorry about that!

**Comment by pomegranited** [comment]
@bmtcril Ooh cool! That's great timing, thank you :)
